client/src/components/PokemonCard/PokemonCard.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Move from '../Move/Move';
+import './style.css';
+import CanvasDraw from 'react-canvas-draw';
+
+const PokemonCard = (props) => {
+
+    /* Changes font color based on the pokemon type */
+    const getStyle = (type) => {
+        switch (type) {
+            case null:
+                return {};
+            case "Normal":
+                return { color: "" };
+            case "Fire":
+                return { color: "#ff00ff" };
+            case "Water":
+                return { color: "#0000ff" };
+            case "Grass":
```

Comment:
Same here with `return { color: colors[type] }` (you should still handle null case)